deploy/charts/rawfile-csi/templates/01-node-plugin.yaml

```diff
@@ -51,7 +51,7 @@ spec:
             type: DirectoryOrCreate
         - name: data-dir
           hostPath:
-            path: /var/csi/rawfile
+            path: "{{ .Values.node.storage.path }}"
```

Comment:
parameterize the `node.storage.path`

Reply:
you can also do `path: {{ .Values.node.storage.path | quote }}`

Reply:
done